docs/filter-functions.md

```diff
@@ -911,3 +912,22 @@ Check if the branch author is a rookie
 ```yaml+jinja
 is_rookie: {{ repo | rankByGitBlame(lt=15) | match(term=branch.author) | some }}
 ```
+
+#### `readFile`
+
+Reads the contents of a file from the current branch or the "cm" folder and returns it as a string. This function only allows reading files from the "repo" and "cm" directories for security reasons.
```

Comment:
same comment, not clear where the file location is expected. I would add an example of filetree and what path should be used